tests/c_languages/CPPLintBearTest.py

```diff
@@ -13,16 +17,42 @@
                                     tempfile_kwargs={"suffix": ".cpp"})
 
 CPPLintBearIgnoreConfigTest = verify_local_bear(
-    CPPLintBear,
-    valid_files=(test_file,),
-    invalid_files=(),
-    settings={'cpplint_ignore': 'legal'},
-    tempfile_kwargs={"suffix": ".cpp"})
+        CPPLintBear,
+        valid_files=(test_file,),
+        invalid_files=(),
+        settings={'cpplint_ignore': 'legal'},
+        tempfile_kwargs={"suffix": ".cpp"})
 
 CPPLintBearLineLengthConfigTest = verify_local_bear(
-    CPPLintBear,
-    valid_files=(),
-    invalid_files=(test_file,),
-    settings={'cpplint_ignore': 'legal',
-              'max_line_length': '13'},
-    tempfile_kwargs={"suffix": ".cpp"})
+        CPPLintBear,
+        valid_files=(),
+        invalid_files=(test_file,),
+        settings={'cpplint_ignore': 'legal',
+                  'max_line_length': '13'},
+        tempfile_kwargs={"suffix": ".cpp"})
+
+
+@generate_skip_decorator(CPPLintBear)
+class CPPLintBearTest(LocalBearTestHelper):
+
+    def setUp(self):
+        self.section = Section("test section")
+        # uut = unit under test
```

Comment:
this is usually considered a common abbreviation. If you argue that it's not common at all we might want to change the name because we use uut in a lot of tests